system/Pager/Pager.php

```diff
@@ -443,6 +443,11 @@ protected function ensureGroup(string $group)
 			'perPage'     => $this->config->perPage,
 			'pageCount'   => 1,
 		];
+
+		if($_GET)
```

Comment:
The tests seem to say this works fine, but I'm curious whether using `addQuery` above and then setting the query below has any chance of duplicating the `page` var?